File: Plugins/Aspose_Tasks_Java_for_Ruby/lib/asposetasksjava/TaskLinks/createtasklink.rb
module Asposetasksjava
  module CreateTaskLink
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate project object
        project = Rjb::import('com.aspose.tasks.Project').new

    	pred = project.getRootTask().getChildren().add("Task 1")
    	succ = project.getRootTask().getChildren().add("Task 2")
    	link = project.getTaskLinks().add(pred, succ)

    	puts "Created task link."
    end
  end
end

File: Plugins/Aspose_Tasks_Java_for_PHP/src/aspose/tasks/WorkingWithTaskLinks/CreateTaskLink.php
<?php
namespace Aspose\Tasks\WorkingWithTaskLinks;

use com\aspose\tasks\Project as Project;
class CreateTaskLink{

    public static function run($dataDir=null){

        # Instantiate project object
        $project = new Project();

        $pred = $project->getRootTask()->getChildren()->add("Task 1");
    	$succ = $project->getRootTask()->getChildren()->add("Task 2");
    	$link = $project->getTaskLinks()->add($pred, $succ);

    	print "Created task link.".PHP_EOL;
    }

}
?>